app/components/ManageCart/reducers/reducer.js

```diff
@@ -74,10 +73,10 @@ export const addToCart = (item) => (dispatch) => {
   .catch(console.error.bind(console))
 }
 
-export const removeFromCart = (item, cartId) => (dispatch) => {
+export const removeFromCart = (item) => (dispatch) => {
   axios.put(`/api/cart/products/sub`, item)
   .then( res => {
-    if (res.data === 'OK') dispatch(removeItem(item))
+    if (res.data === 'OK') dispatch(retrieveItems())
```

Comment:
Dispatching retrieveItems is a clever workaround for effectively using the backend logic for removing an item!  
I was actually pondering this problem when I looked at this, and I had never thought of that solution. 

However, it does render completely moot the REMOVE_ITEM action creator, reducer case, and constant.